.github/workflows/main.yml

```diff
@@ -96,11 +96,19 @@ jobs:
             rust: stable
             target: aarch64-apple-ios
             no_run: --no-run
-          - build: windows-aarch64
+          - build: cross-windows-aarch64
             os: windows-latest
             rust: stable
             target: aarch64-pc-windows-msvc
             no_run: --no-run
+          - build: windows-aarch64
+            os: windows-11-arm
+            rust: stable
+            target: aarch64-pc-windows-msvc
+          - build: cross-win64
+            os: windows-11-arm
+            rust: stable
+            target: x86_64-pc-windows-msvc
```

Comment:
GitHub Actions has Windows 11 ARM runners nowadays, so let's leverage them to build and test on native arm64 hosts.